services/Skyra.Database/Models/Entities/Guild.cs

```diff
@@ -68,25 +68,49 @@ public class Guild
 		[StringLength(19)]
 		public string? ChannelsSpam { get; set; }
 
-		[Column("channels.logs.member")]
+		[Column("channels.logs.moderation")]
 		[StringLength(19)]
-		public string? ChannelsLogsMember { get; set; }
+		public string? ChannelsLogsModeration { get; set; }
```

Comment:
That's because `channels.logs.member` was removed, and so was `channels.logs.message`. Since the line is very similar to the one for `channels.logs.moderation`, Git Diff shows a misleading change.